### DISCUSSION TITLE
auto concede bot waits too long on turn 4 to concede

### DISCUSSION THREAD

**Discussion by Dice17** [AdriaGual/marvel-snap-bot#104] [General]
As the title says, may i know how to improve that?

**Comment by AdriaGual** [comment]
You can improve the detection algorithm to use AI insted of CV and detect turn 4 faster.

**Reply by Dice17** [reply]
i think it is because of the images, because the bot always retreat turn 4 at a specific time, but when i added my own screenshot it doesn't seems to detect it, is there any quality or size requirements for the screenshot in order for it to work?

**Reply by paperc07** [reply]
Yes

**Reply by Dice17** [reply]
can i know the details so i can add it myself?
edit : nevermind i had already solve it, thank you!

For those who wondering, you need to use bluestacks to take the screenshot and crop it

**Reply by Nerdstrong** [reply]
have you been able to use the auto concede with success and without suspension?

**Reply by Dice17** [reply]
i've been using it for days and i haven't encountered any suspension so far, probably will stop using it if i get suspended

**Reply by Dice17** [reply]
okay i just got suspended, so try to avoid using this

**Reply by paperc07** [reply]
Yeah I lasted what I would say the longest and I wasn't mad about it I got to 125 this season and it got me to rank 56 and I pushed to 83 so I'll just wait for something else to come along and or another bot

**Reply by Dice17** [reply]
I got from 70 to 135 in like 4 days of using it, i run for almost 24 hours, maybe that's the reason why i got suspended so quickly

**Reply by paperc07** [reply]
yeah thats what I did lol
